apps/web_delivery/lib/web_delivery/application.ex

```diff
@@ -0,0 +1,21 @@
+defmodule WebDelivery.Application do
+  use Application
+
+  alias WebDelivery.Endpoint
+
+  ## Callbacks
+
+  def start(_type, _args) do
+    import Supervisor.Spec
+
+    Supervisor.start_link [supervisor(Endpoint, [])],
+      strategy: :one_for_one,
+      name: WebDelivery.Supervisor
+  end
+
+  @spec config_change(keyword, keyword, [atom]) :: :ok
+  def config_change(changed, _new, removed) do
+    Endpoint.config_change(changed, removed)
+    :ok
+  end
+end
```

Comment:
It’s cleaner, in general, to have your application callbacks in a dedicated module, but the more I think about it I doubt that any functions will be defined directly under the `WebDelivery` namespace. I’m actually going to merge the callbacks back in.